test/findTest.js

```diff
@@ -0,0 +1,61 @@
+var db = require('../db')
+var fs = require('fs')
+var async = require('async')
+var collectionName, result
+var done = function(err, resultArray) {result = resultArray}
+var queryFnTrue = function() {return true}
+var queryFnFalse = function() {return false}
+var expect = require('chai').expect
+
+var soldierObj1 = {
+    id: 1,
+    name: "Noa",
+    rank: "corporal",
+    limitations: "none",
+    duties: "1"
+
+}
+
+fs.mkdirSync("./collections/soldierCollection1", { recursive: true }, function(err) {
+    if (err) {
+      console.log(err)
+    } else {
+      console.log("first directory successfully created.")
```

Comment:
console logs are great for debuging in development, but for a user using these packages they are noise. remove all console logs from the commits themselves. you can implement a logger that would log to file if you wish but please don't though